src/lib/templateHelpers.ts

```diff
@@ -4,35 +4,66 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+
 import { ASSET_OBJECT_MAP } from "./config";
 
 import { commonParameterPositions } from "@commerce-apps/core";
 
-import _ from "lodash";
-
 /**
- * Checks if a path parameter is one of the set that are configurable at the client level
- *
- * @param property - The string name of the parameter to check
+ * Given an individual type or an array of types in the format Array\<Foo | Baa\>
+ * will return either the type prefixed by the namespace, or the Array with each type prefixed
+ * eg. Array\<types.Foo | types.Baa\>
  *
- * @returns true if the parameter is a common parameter
+ * @param content - to be parsed for types to prefix with a namespace
+ * @param namespace - to be prefixed to types
+ * @returns the content prefixed with the namespace
  */
-export const isCommonPathParameter = (property: string): boolean =>
-  property
-    ? commonParameterPositions.pathParameters.includes(property.toString())
-    : false;
+export function addNamespace(content: string, namespace: any): string {
+  // Not handling invalid content.
+  if (!content) {
+    throw new Error("Invalid content");
+  }
+  // Not handling invalid namespace.
+  if (!namespace) {
+    throw new Error("Invalid namespace");
+  }
 
-/**
- * Checks if a query parameter is one of the set that are configurable at the client level
- *
- * @param property - The string name of the parameter to check
- *
- * @returns true if the parameter is a common parameter
- */
-export const isCommonQueryParameter = (property: string): boolean =>
-  property
-    ? commonParameterPositions.queryParameters.includes(property.toString())
-    : false;
+  // if the content is an array, extract all of the elements
+  const matched = content.match(/^Array<(.*?)>$/);
+  const arrayType = !!matched;
+  const types = matched?.[1] || content;
```

Comment:
potential index out of bounds error.. do u want to check if the length is greater than 1 before getting the index 1

Reply:
This was also copied over.  This might be an issue but we should confirm.  Also written by @dbarbato-sf

Reply:
JavaScript doesn't throw if you access an unset index, it just returns `undefined`. This is safe.